frontend/src/store/cloudProfile/index.js

```diff
@@ -297,6 +297,20 @@ export const useCloudProfileStore = defineStore('cloudProfile', () => {
     return machineTypesOrVolumeTypesByCloudProfileNameAndRegion({ type: 'machineTypes', cloudProfileName })
   }
 
+  function expirationWarningSeverityForVersion ({ isExpirationWarning, autoPatchEnabled, updateAvailable, autoUpdatePossible }) {
+    if (isExpirationWarning) {
+      if (!updateAvailable) {
+        return 'error'
+      } else if ((!autoPatchEnabled && autoUpdatePossible) || !autoUpdatePossible) {
+        return 'warning'
+      } else {
+        return 'info'
+      }
+    } else if (autoPatchEnabled && autoUpdatePossible) {
+      return 'info'
+    }
+  }
```

Comment:
```suggestion
  function getVersionExpirationWarningSeverity (options) {
    const { 
      isExpirationWarning,
      autoPatchEnabled,
      updateAvailable,
      autoUpdatePossible,
    } = options
    const autoPatchEnabledAndPossible = autoPatchEnabled && autoUpdatePossible
    if (!isExpirationWarning) {
      return autoPatchEnabledAndPossible 
        ? 'info' 
        : undefined
    } 
    if (!updateAvailable) {
      return 'error'
    } 
    return autoPatchEnabledAndPossible 
        ? 'info'
        : 'warning'
    }
  }
```
The name `expirationWarning` is very confusing. The name implies it is a warning and not an error or an info. This should be  `expirationMessage` or `expirationNotification`.